test/tools/OpenCover/OpenCover.psm1

```diff
@@ -40,6 +40,172 @@ function Get-ClassCoverageData([xml.xmlelement]$element)
+    }
+}
+
+<#
+.Synopsis
+   Format the coverage data for a file
+.Description
+   Show the lines which were hit or not in a specific file. Line numbers are included in the output.
+   If a line was hit during a test run a '+' will follow the line number, if a line was missed, a '-'
+   will follow the line number. If a line is not hittable, it will not show '+' or '-'.
+
+   You can map file locations with the -oldBase and -newBase parameters (see example below), so you can 
+   view coverage on a system with a different file layout. It is obvious to note that if files are different
+   between the systems, the results will be misleading
+.EXAMPLE
+   PS> $coverage = Get-CodeCoverage -CoverageXmlFile .\opencover.xml
+   PS> Format-FileCoverage -FileCoverageData $coverage.FileCoverage -filter "CredSSP.cs"
+
+   ...
+   0790                   try
+   0791 +                 {
+   0792                       // ServiceController.Start will return before the service is actually started
+   0793                       // This API will wait forever
+   0794 +                     serviceController.WaitForStatus(
+   0795 +                         targetStatus,
+   0796 +                         new TimeSpan(20000000) // 2 seconds
+   0797 +                         );
+   0798 +                     return true; // service reached target status
+   0799                   }
+   0800 -                 catch (System.ServiceProcess.TimeoutException) // still waiting
+   0801 -                 {
+   0802 -                     if (serviceController.Status != pendingStatus
+   ...
+
+.EXAMPLE
+   Map the file location from C:\projects\powershell-f975h to /users/james/src
+   PS> $coverage = Get-CodeCoverage -CoverageXmlFile .\opencover.xml
+   PS> $formatArgs = @{
+       FileCoverageData = $coverage.FileCoverage
+       filter = "Service.cs"
+       oldBase = "C:\\projects\\powershell-f975h"
+       newBase = "/users/james/src"
+   }
+   PS> Format-FileCoverage @formatArgs
+
+   ...
+   0790                   try
+   0791 +                 {
+   0792                       // ServiceController.Start will return before the service is actually started
+   0793                       // This API will wait forever
+   0794 +                     serviceController.WaitForStatus(
+   0795 +                         targetStatus,
+   0796 +                         new TimeSpan(20000000) // 2 seconds
+   0797 +                         );
+   0798 +                     return true; // service reached target status
+   0799                   }
+   0800 -                 catch (System.ServiceProcess.TimeoutException) // still waiting
+   0801 -                 {
+   0802 -                     if (serviceController.Status != pendingStatus
+   ...
+#>
+function Format-FileCoverage
+{
+    param ( 
+        [Parameter(Position=0,Mandatory=$true,ValueFromPipeline=$true)]$FileCoverageData, 
+        [Parameter()][string]$oldBase = "", 
+        [Parameter()][string]$newBase = "", 
```

Comment:
see above - generally don't want to change the path, but sometimes I do. setting this to "" makes it work nearly all the time, but when I'm working on my mac which has different path structure, I can still look at the coverage data. In this case I may not be in a repo at all.